src/utils/external-extensions.ts

```diff
@@ -105,3 +175,5 @@ export const getSolidityFrameworkDirsFromExternalExtension = async (
 
   return filterSolidityFrameworkDirs(directories);
 };
+
+export { CURATED_EXTENSIONS, TRUSTED_GITHUB_ORGANIZATIONS };
```

Comment:
NIT: Shall we export this variable like other functions? Instead of doing export at bottom of file?

Reply:
I actually forgot to remove it! Not exporting them since they are not being used anywhere outside this file 

Thanks!

447a07762e08fe861cc0064677619e48d72bf590